packages/nx/src/command-line/nx-commands.ts

```diff
@@ -53,7 +53,7 @@ export const parserConfiguration: Partial<yargs.ParserConfigurationOptions> = {
  * from the `.argv` call, so the object and it's relative scripts can
  * le executed correctly.
  */
-export const commandsObject = yargs
+export const commandsObject: any = yargs
   .parserConfiguration(parserConfiguration)
```

Comment:
Type check for any

Reply:
Please revert this, I'm not sure why it was added

Reply:
Updated for CI check `pnpm run documentation`

Reply:
This again must be something unusual about your setup. There is no known general issue with this code on master, nor on mine or other's machines. Please revert this change and let's see what happens in CI

Reply:
Either that or please link me to a CI run where this failed, I cannot see one